### ISSUE TITLE
[Chests Anywhere] add option to edit chest from default UI

### ISSUE THREAD

**Issue by Pathoschild** [Pathoschild/StardewMods#43]
**Imported from Pathoschild/ChestsAnywhere#1. Original date: 2016-10-05 22:04 UTC.**
<hr />

The mod lets you edit chests through the Chests Anywhere screen:

![edit chest UI](https://cloud.githubusercontent.com/assets/230581/22623971/75eb291c-eb3a-11e6-8c15-79aab339f6de.png)

...but there's no way to edit a chest from the game's default UI:
![view chest](https://cloud.githubusercontent.com/assets/230581/22623974/81c531ba-eb3a-11e6-8b11-e3d9f4cecb65.png)

Add a button to the default UI to access the chest's edit screen.

[Discussed in the forums](http://community.playstarbound.com/threads/chest-label-system.112420/page-6#post-3034960).

**Comment by Pathoschild** [comment]
**Imported from Pathoschild/ChestsAnywhere#1. Original date: 2016-10-07 05:20 UTC.**
<hr />

Done in the upcoming 1.5 release (note the little icon over the chest):
![sample chest UI](https://cloud.githubusercontent.com/assets/230581/22623980/02151920-eb3b-11e6-8e47-3be5d7afa615.png)

This uses a generic overlay system introduced by d0d24eab1bd37c44427c94b8e2da23f17f8fa159, which floats components on top of the existing UI (including the active clickable menu). Overlays are self-perpetuating components which can receive clicks, draw themselves, etc. An overlay can have a keep-alive lambda, which destroys the overlay when it stops returning true (e.g. if you only want to float a component as long as a particular menu is open).

tl;dr: extending existing UI for future features is much easier now.
